ReactNativeClient/lib/joplin-renderer/MdToHtml/rules/copy_to_clipboard.js

```diff
@@ -0,0 +1,137 @@
+const copyStyle = `
+.code-n-btn-wrapper {
+  positon: relative;
```

Comment:
"positon" is not a valid CSS property, and this is why your class "copy-code-wrapper" didn't work and you needed the hacky margins.

Reply:
Thanks for showing me this!